pootle/apps/pootle_store/receivers.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from .models import Unit
+
+
+@receiver(post_save, sender=Unit)
+def unit_postsave_handler(**kwargs):
```

Comment:
@phlax sorry missed the last part of PR description.
In this case store.update() should be wrapped within transaction, as I can understand.